src/Amp/views/apache-vhost.php

```diff
@@ -5,12 +5,19 @@
  * @var string $host - the hostname to listen for
  * @var int $port - the port to listen for
  * @var string $include_vhost_file - the local path to a related config file
+ * @var string $visibility - which interfaces the vhost is available on
  */
 ?>
 
 <?php if ($use_listen) { ?>
 
+<?php if ($visibility === 'all'): ?>
 Listen <?php echo $port ?>
+<?php else: ?>
+Listen 127.0.0.1:<?php echo $port ?>
+
+Listen [::1]:<?php echo $port ?>
```

Comment:
yes, IPv6 could be disabled using sysctl.  In this case then apache would fail to start.   I am guessing it's (currently :) far more common to disable IPv6 than to disable IPv4 so i'll tweak this to only listen on 127.0.0.1